doc/source/reference/c-api/data_memory.rst

```diff
@@ -50,7 +50,7 @@ management routines can use :c:func:`PyDataMem_SetHandler`, which uses a
 :c:type:`PyDataMem_Handler` structure to hold pointers to functions used to
 manage the data memory. The calls are still wrapped by internal routines to
 call :c:func:`PyTraceMalloc_Track`, :c:func:`PyTraceMalloc_Untrack`, and will
-use the deprecated :c:func:`PyDataMem_EventHookFunc` mechanism. Since the
+use the deprecated ``PyDataMem_EventHookFunc`` mechanism. Since the
```

Comment:
I think this should be deleted, and we should remove this functionality for NumPy2.0. @seberg: thoughts?

Reply:
We discussed this and decided to remove the hook. Please remove the reference.

Reply:
Actually, the function has already been removed for NumPy2.0. I think `PyDataMem_EventHook` also appears elsewhere in this file, could you remove that as well?

Reply:
All removed, `PyDataMem_EventHookFunc` and `PyDataMem_EventHook`.